stan/math/prim/mat/prob/ordered_logistic_lpmf.hpp

```diff
@@ -55,8 +58,13 @@ namespace stan {
      */
     template <bool propto, typename T_lambda, typename T_cut>
     typename boost::math::tools::promote_args<T_lambda, T_cut>::type
-    ordered_logistic_lpmf(int y, const T_lambda& lambda,
-                         const Eigen::Matrix<T_cut, Eigen::Dynamic, 1>& c) {
+    ordered_logistic_lpmf(int y,
+                      const T_lambda& lambda,
```

Comment:
We'd rather keep as many arguments as possible on a line.

The running theme here is conserving vertical space so it's easier to scan a program.